drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -45,7 +45,8 @@ function DragNDropTemplates(url_name) {
             'draggable': !item.drag_disabled,
             'aria-grabbed': item.grabbed,
             'data-value': item.value,
-            'data-drag-disabled': item.drag_disabled
+            'data-drag-disabled': item.drag_disabled,
+            'aria-describedby': gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.'),
```

Comment:
@dovinmu I'm not actually sure there is one, just wanted you to be aware of that technique in case it was a viable solution. I don't want to make any suggestions for implementation techniques until some manual testing is performed on this. Much of accessibility is pretty straightforward, but we are getting into an area that has very poor implementation support, so a lot of testing and revision work is required.  I'm hoping to find the time to take a close look at this soon. I don't want to test this though until we at least get an IDref in there as the value of that aria-describedby attribute.